examples/00-EDB/02_edb_to_ipc2581.py

```diff
@@ -45,6 +45,31 @@
     "A0_N", parameter_name=generate_unique_name("Par"), variable_value="0.4321mm"
 )
 
+###############################################################################
+# Cutout
+# ~~~~~~
+# Create a cutout using new Pyaedt method.
```

Comment:
```suggestion
# Create a cutout using a PyAEDT method.
```
We want timeless documentation, which means not referring to "new" or "old" in descriptions. I'm also not sure we need the "using a PyAEDT method" portion of the sentence since it's all about using PyAEDT.